domain-model/src/main/java/uk/gov/hmcts/cmc/domain/amount/TotalAmountCalculator.java

```diff
@@ -29,11 +29,15 @@ private TotalAmountCalculator() {
     }
 
     public static Optional<BigDecimal> totalTillToday(Claim claim) {
-        return Optional.ofNullable(calculateTotalAmount(claim, LocalDate.now()));
+        return Optional.ofNullable(calculateTotalAmount(claim, LocalDate.now(), true));
+    }
+
+    public static Optional<BigDecimal> amountWithInterest(Claim claim) {
```

Comment:
Can you move it above `totalTillToday` so that methods for total amount are grouped together.